src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +25,26 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws FileNotFoundException {
+        StringBuilder text = new StringBuilder();
+        try (Scanner scanner = new Scanner(new FileReader(fileName))) {
+            while (scanner.hasNextLine()) {
+                text.append(scanner.nextLine().toLowerCase().replaceAll("[^a-z]", ""));
+            }
+        } catch (IOException e) {
+            throw new FileNotFoundException("Mentioned file does not exist");
+        }
```

Comment:
you are throwing checked Exception from this method anyway and indicate this in method signature. try just removing this catch block